pkgs/applications/audio/spotify/default.nix

```diff
@@ -23,7 +23,7 @@ let
     curl
     dbus
     expat
-    ffmpeg_0_10
+    ffmpeg
```

Comment:
Does this really work (can local files be played)? Arch package requires https://aur.archlinux.org/packages/ffmpeg-compat-54/